pom.xml

```diff
@@ -214,7 +213,7 @@
         <resource-server.version>1.0.41</resource-server.version>
         <servlet-api.version>3.0.1</servlet-api.version>
         <slf4j.version>1.7.7</slf4j.version>
-        <spring-framework.version>3.1.4.RELEASE</spring-framework.version>
+        <spring-framework.version>4.0.5.RELEASE</spring-framework.version>
```

Comment:
As Gary R mentioned in the PR description, there was a change made in Spring Framework that breaks later versions of Spring Framework 3.1, 4.0, and all 4.1.  See the link in the description.  This PR is an interim step toward newer versions of Spring Framework that would at least get us to Java 8.